.travis.yml

```diff
@@ -7,6 +7,8 @@ install:
   - composer install
   - bash tests/install-tests.sh wordpress_test root '' 127.0.0.1 latest
 script:
-  - vendor/bin/phpunit --exclude-group cron
+  # 32656 tests are related to the pre_*_event filters which we're using already
+  # 45976 tests compare event objects directly, cavalcade adds the _job property
```

Comment:
In #94 I've removed the job from the returned jobs as there's no backcompat issues that it will raise with the plugin. Using the `pre_option` hook hack, getting a scheduled event doesn't return the `_job` property. Your call though.